src/opening-period/form/OpeningPeriodForm.tsx

```diff
@@ -278,46 +282,53 @@ export default function OpeningPeriodForm({
             <h3 className="opening-period-section-title">Ajanjakso</h3>
             <div className="form-control">
               <section className="opening-period-time-period">
-                <DateInput
-                  id="openingPeriodBeginDate"
+                <Controller
```

Comment:
The DateInput onChange was incompatible with the one returned from the register.

Reply:
Probably could have done some sort of mapping in the register.